apps/client/src/pages/dashboard/apis/userApi.ts

```
@@ -0,0 +1,22 @@
+  now?: string
+): Promise<ApiResponse<AcornCountWithRemindResponse>> => {
+  try {
+    const url = now
+      ? `/api/v1/users/acorns?now=${encodeURIComponent(now)}`
+      : '/api/v1/users/acorns';
+
+    const { data } =
+      await apiRequest.get<ApiResponse<AcornCountWithRemindResponse>>(url);
+    return data;
+  } catch (error: any) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)